src/opera/pge/rtc_s1/rtc_s1_pge.py

```diff
@@ -710,6 +716,40 @@ def _collect_rtc_product_metadata(self, metadata_product):
         output_product_metadata['data']['length'] = len(output_product_metadata['data']
                                                         ['yCoordinates'])
 
+        product_type = self.runconfig.product_type
+
+        # If creating ISO metadata for a static layer product, we need to
+        # formulate the bounding box, rather than the bounding polygon
+        if product_type == "RTC_S1_STATIC":
+            bounding_box = output_product_metadata['identification']['boundingBox']
+            epsg_code = int(output_product_metadata['data']['projection'])
+
+            lat_min, lat_max, lon_min, lon_max = translate_utm_bbox_to_lat_lon(bounding_box, epsg_code)
+
+            output_product_metadata['staticBoundingBox'] = {
+                'lonMin': lon_min,
+                'latMin': lat_min,
+                'lonMax': lon_max,
+                'latMax': lat_max
+            }
+
+        # We also need to assign the URL for static layer data access for
```

Comment:
Again, the comment (smiling).  This statement ending makes me thing 'as opposed to ......'